exe/sufia_survey

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+require 'rails'
+
+require ::File.join(File.dirname(Bundler.default_gemfile), 'config/environment')
+
+options = {}
+
+subtext = <<HELP
+Commonly used command are:
+   clean :     cleans all Sufia::Migration::Survey::Items from the database
+See 'sufia_survey COMMAND --help' for more information on a specific command.
+HELP
+
+args = OptionParser.new do |opts|
+  opts.banner = "Survey model objects in fedora and store it in an ActiveRecord table. Usage: sufia_survey [options]"
+
+  opts.on("-v", "--[no-]verbose", "Run verbosely") do |v|
```

Comment:
So `--no-verbose` and `--verbose` both trigger verbose output? Maybe I misunderstand what OptionParser is doing here.